src/utility/mocks.ts

```diff
@@ -0,0 +1,76 @@
+import { assert } from "chai";
```

Comment:
mocks are definitely not utilities. Move to `test/mocks` folder and (ideally) split by mock type

Reply:
Will move once the other PR is merged to prevent conflicts